### DISCUSSION TITLE
Issue with opening new Connection Tabs using the  +  button

### DISCUSSION THREAD

**Discussion by ghost** [Sequel-Ace/Sequel-Ace#1822] [General]
I've noticed a bug (which is still present in the latest Mac AppStore version) and wanted to report it as it's nearly meant me losing data! I will try to explain it. I generally have multiple databases open in Connection Tabs - for simplicity let's call them A, B, C, D, E. Now, i f I need to connect to 'F' then I can press the '+' button at the right hand side of the and connect to the new database - but this only works correctly if you have the last tab selected. If I am working on 'C' and click the '+' then it adds 'F' to the right of 'C' but the tab title says 'D' (but the connection details at the top say it's 'F'). The 5th tab should be 'D' (but shows 'E'), and the 6th should be 'E' (but shows 'F'). So if I want to work on 'F' I actually need to click on the tab that says 'D'. This stays the same if I move the tabs around and it is very confusing. If, however, I use the menu to open a new tab (or press CMD+T) then the tab is added correctly regardless of what tab is initially selected. It only seems to affect the tab's titles and if you're using the '+' button on the tab's row. Hopefully this makes sense!

**Comment by Jason-Morcos** [comment]
See this existing issue - https://github.com/Sequel-Ace/Sequel-Ace/issues/1797
I believe @Kaspik is looking into this :)
